lib/cli.js

```diff
@@ -543,6 +543,53 @@ yargs
             Mcdev.fixKeys(argv.BU, argv.TYPE, csvToArray(argv.KEY));
         },
     })
+    .command({
+        command: 'updateNotifications <BU> <TYPE> [KEY]',
+        aliases: ['updNotif'],
+        desc: 'bulk updates notification email addresses',
+        builder: (yargs) => {
+            yargs
+                .positional('BU', {
+                    type: 'string',
+                    describe: 'the business unit where to update notification email address',
+                })
+                .positional('TYPE', {
+                    type: 'string',
+                    describe: 'metadata type',
+                })
+                .positional('KEY', {
+                    type: 'string',
+                    describe: 'key(s) of the metadata component(s)',
+                })
+                .positional('emailError', {
+                    type: 'string',
+                    describe: 'email to notify that an error occured during execution',
+                })
+                .positional('emailComplete', {
+                    type: 'string',
+                    describe: 'email to notify about successfull completion',
+                })
+                .positional('errorNote', {
+                    type: 'string',
+                    describe: 'run error note',
+                })
+                .positional('completionNote', {
```

Comment:
lets turn this into an option - better fit for the different requirements of the types because otherwise you couldn't "skip" a parameter: imagine somebody only wants completion notifications for automation with a custom note.